### PR TITLE
Notify internal Slack of customer Oauth of Slack and Gmail - Issue #282

### PR DESCRIPTION
**Pull request by lorecrafting**
### Description

Notify's Papercups internal Slack if a customer performs an oauth flow on Slack and Gmail.

### Issue

_https://github.com/papercups-io/papercups/issues/282_

## Checklist

- [x] Everything passes when running `mix test`
- [x] Ran `mix format`
- [x] No frontend compilation warnings

### CODE REVIEW

**Comment by lorecrafting** [comment]
I haven't tested it with the #sandbox channel yet, as i'm getting redirect_uri errors.

Would you please let me know if what I currently have is in the right direction? Thanks!

File: lib/chat_api_web/controllers/gmail_controller.ex [hunk 1]
```diff
@@ -75,6 +75,7 @@ defmodule ChatApiWeb.GmailController do
       )
       |> case do
         {:ok, result} ->
+          notify_slack(conn)
```
Comment (reichert621) [inline lib/chat_api_web/controllers/gmail_controller.ex:78]: we can also pipe this with the line below :)
```elixir
conn |> notify_slack() |> json(%{ok: true, data: result})
```

**Comment by reichert621** [comment]
@lorecrafting looks fine so far -- just one suggestion :) 

sorry this will be a little difficult to test, so feel free to just make the update suggested above and then i can handle testing 👍 

also, can you update the PR title to something more descriptive? thanks!

File: lib/chat_api_web/controllers/gmail_controller.ex (hunk 1 above)
Comment (lorecrafting) [reply lib/chat_api_web/controllers/gmail_controller.ex]: Done!

**Review by reichert621 — APPROVED** [review]
thanks @lorecrafting!

**Pull request merged**